internal/ethapi/api.go

```diff
@@ -321,6 +324,9 @@ func (s *PrivateAccountAPI) Sign(ctx context.Context, message string, addr commo
 // hash = keccak256("\x19Ethereum Signed Message:\n"${message length}${message})
 // addr = ecrecover(hash, signature)
 //
+// Note, the signature must conform to the Ethereum yellow paper spec where the V
+// parameter must be 27 or 28 irrelevant of any ChainID features.
```

Comment:
No need to mention ChainID. It's sufficient to say that V must be 27 or 28 here for legacy reasons.